src/components/StartMenu/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import './modal-window.css';
+
+const ModalWindow = (props) => {
+    const { handleComputerMode, handleChoosePlayer, player } = props;
+    return (
+        <>
+            <div className="modal-window-overlay"></div>
+            <div className="modal-window">
+                <div>
+                    Choose your player:
+                    <span className="players">
+                        <label className={player === 'X' ? "active-player" : ''} htmlFor="checkX">X</label>
+                        <input type="radio" name="player" value='X' id="checkX" onChange={(e) => handleChoosePlayer(e, e.target.value)} />
+                        <label className={player === 'O' ? "active-player" : ''} htmlFor="checkO">O</label>
+                        <input type="radio" name="player" value='O' id="checkO" onChange={(e) => handleChoosePlayer(e, e.target.value)} />
```

Comment:
The handler will have access to the event. You can destructure the data there and write simply:
`onChange={handleChoosePlayer}`